### PR TITLE
CBSP-815: Allow passing of custom URL

### PR DESCRIPTION
**Pull request by JFlath**

### CODE REVIEW

File: puppet.pp [hunk 1]
```diff
@@ -9,7 +9,13 @@
     OpenSuSE => ".rpm",
 }
 
-$fullUrl = "$url$suffix"
+# Doublecheck this, as url may already include this if passed by ENV
+if $suffix in $url {
```
Comment (malarky) [inline puppet.pp]: Is this OK?  Did you test the different flavours?

Comment (JFlath) [inline puppet.pp]: Grr, making me pull Debian boxes :(

Comment (JFlath) [inline puppet.pp]: Heh, as it happens, it looks like out debian boxes are broken on an up to date vagrant. Perhaps we need to workaround that with the box_version as for centos...

Comment (malarky) [inline puppet.pp]: :woman_facepalming: 

I can test it probably...

**Pull request merged**